apps/common-app/src/examples/RuntimeTests/RuntimeTestsExample.tsx

```diff
@@ -75,6 +75,12 @@ export default function RuntimeTestsExample() {
             require('./tests/plugin/contextObjects.test');
           },
         },
+        {
+          testSuiteName: 'StrictMode',
+          importTest: () => {
+            require('./tests/StrictMode/StrictMode.test');
+          },
+        },
```

Comment:
One of the main goals of splitting the tests into sections was to avoid tests running for too long, like more than 15 minutes as the simulator or computer could start sleep mode, which have negative impact on tests. Could we have this sections more uniform when considering duration? For example could you merge StrictMode with some other section and rename it into sth like "other"? I am open for discussion on how we want to manage the scope and number of sections.